### ISSUE TITLE
API error

### ISSUE THREAD

**Issue by szepeviktor** [WPCentral/WP-CLI-Security#3]
https://wpcentral.io/api/checksums/plugin/akismet/3.1.11

``` json
{"code":"wpcentral_server_error","message":"Generating checksums","data":{"status":202}}
```

`Fatal error: Cannot use object of type stdClass as array in ~/.wp-cli/commands/WP-CLI-Security/checksum.php on line 52`

**Comment by markoheijnen** [comment]
Checksums aren't returned the first time. Second request will return the correct data.
I still need to work that everything is already in the database.

**Comment by szepeviktor** [comment]
Actually the n-th result is also `wpcentral_server_error`.

**Comment by szepeviktor** [comment]
Non-WP.org plugins also cause a Fatal error. E.g. `sensei-badgeos`
